src/queue.js

```diff
@@ -122,12 +120,25 @@ function Queue() {
     logger.debug('Queue(): Error during initialization', error);
     throw new Error(error);
   }
+  
+  if (_.isPlainObject(constructorArguments[0])) {
+    if (_.isUndefined(constructorArguments[0].tasksRef) || _.isUndefined(constructorArguments[0].specsRef)) {
+      error = 'When queueRef is an object then it must specify tasksRef and specsRef.';
```

Comment:
Grammar: `When queueRef is an object it must contain both a tasksRef and a specsRef`